test/test.js

```diff
@@ -0,0 +1,49 @@
+let createS3UploadParams = require('../index').createS3UploadParams
+let reverse = require('../index').reverse
+let assert = require('assert');
+
+describe('Reverse', function () {
+  const tests = [
+    { word: 'abc', expected: 'cba' }
+  ];
+
+  tests.forEach(({ word, expected }) => {
+    it(`reverses ${word} correctly`, function () {
+      assert.equal(reverse(word), expected);
+    });
+  });
+});
+
+describe('CreateS3UploadParams', function () {
+  ip_ranges_example = {
+    "syncToken": "0123456789",
+    "createDate": "yyyy-mm-dd-hh-mm-ss",
+    "prefixes": [
+      {
+        "ip_prefix": "cidr",
+        "region": "region",
+        "network_border_group": "network_border_group",
+        "service": "subset"
```

Comment:
Codacy has a fix for the issue: [Replace `················"service":·"subset"` with `········service:·"subset",`](https://app.codacy.com/gh/dmgress/aws-ip-ranges/pullRequest?prid=9261990)
```suggestion
        service: "subset",
```